contributing-guides/style-guide.md

```diff
@@ -4,40 +4,113 @@ This page lists specific formatting instructions for `tldr` pages.
 
 ## Layout
 
-The basic format of each page should match the following template:
+> :bulb: Placeholder syntax is not used in this chapter, but instead of:
+>
+> - `command_name`
+> - `https://example.com/command-name/help/page`
+> - `Code description`
+> - `first_options`
+> - `second_options`
+> - `page_name`
+>
+> you can use any valid value as they are placeholders.
```

Comment:
I am confused as to what this part even is, could you explain

Reply:
It is a note about that `{{`, `}}` are not used here explicitly in some cases.